tests/e2e/specs/admin/deposits.spec.js

```diff
@@ -1,25 +1,19 @@
 /**
  * External dependencies
  */
-import config from 'config';
-
 const { merchant } = require( '@woocommerce/e2e-utils' );
 
-const WCPAY_DEPOSITS =
-	config.get( 'url' ) +
-	'wp-admin/admin.php?page=wc-admin&path=/payments/deposits';
+/**
+ * Internal dependencies
+ */
+import { merchantWCP } from '../../utils/flows';
 
 describe( 'Admin deposits', () => {
 	beforeAll( async () => {
 		await merchant.login();
 	} );
 
 	it( 'page should load without any errors', async () => {
-		await page.goto( WCPAY_DEPOSITS, {
-			waitUntil: 'networkidle0',
-		} );
-		await expect( page ).toMatchElement( 'h2', {
-			text: 'Deposit history',
-		} );
+		await merchantWCP.openDeposits();
```

Comment:
Hi @reykjalin, thanks for reviewing!
There's already expect inside every `openDisputes`, `openDeposits` and `openTransactions` so there will be assertions when executed. I hope this answers your question.

Please let me know if I can merge this PR or you'll be merging it, don't know the procedure.

Reply:
> There's already expect inside every openDisputes, openDeposits and openTransactions so there will be assertions when executed.

Yep, I saw that, but that's not what I was trying to get across 🙂 

I mean that the readability of the test is compromised by the fact that there is no _visible_ assertion in the test itself. It just reads like you're calling a function, which feels weird.

I understand the utility of having the assertion inside the `openX` function calls, but I think we should either:
- rename the `openX` functions in such a way that it is clear that an assertion is happening in that function; or 
- have the `openX` functions return a value we can perform an assertion on.

Please refer back to the examples in my previous comment for pseudo-ish code showing what I mean by those 2 points.

I'll repeat my last sentence from the first comment; I won't require this change because I understand the benefits. I just want to make sure we've fully thought this naming convention and pattern through before we merge it in.

> Please let me know if I can merge this PR or you'll be merging it, don't know the procedure.

Once we've finished this discussion and I've approved the PR through GitHub's UI you can merge 🙂

Reply:
I understand @reykjalin and I did changes now.

I moved the assertion outside of the function so it's visible now in the test just below the function for opening the page. I always expect to see assertion when the function says `openSomething`. I think now its good because the assertion is visible properly in the test and we don't need it inside the function. Feel free to merge if you approve it or I'll do that otherwise, thanks!

Reply:
Thank you @veljkho!

This is looking good 🎉  I'll approve the PR and you can go ahead and merge 🚀